README.md

```diff
@@ -19,7 +19,9 @@ Tested with the current Ansible 2.9 and 2.10 releases and the current developmen
 - [digital_ocean_domain](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digital_ocean_domain_module.html) – Create/delete a DNS domain in DigitalOcean
 - [digital_ocean_domain_facts](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digital_ocean_domain_facts_module.html) – Gather information about DigitalOcean Domains
 - [digital_ocean_domain_info](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digital_ocean_domain_info_module.html) – Gather information about DigitalOcean Domains
+- [digital_ocean_domain_record](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digital_ocean_domain_record.html) – Create and delete DigitalOcean Domain Records
```

Comment:
Not sure whether you want to also list the inventory plugin in this section. Besides that, looks ready for release!

Reply:
> Not sure whether you want to also list the inventory plugin in this section. Besides that, looks ready for release!

I'd like to, but, I'm not sure what it'd look like (I don't understand the mechanism behind docs.ansible.com), would you mind helping?

Reply:
I have a feeling it's more than simply adding this?

```
- [digitalocean](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digitalocean.html) – DigitalOcean Inventory Plugin
```

Reply:
Almost:
```
- [digitalocean](https://docs.ansible.com/ansible/2.10/collections/community/digitalocean/digitalocean_inventory.html) – DigitalOcean Inventory Plugin
```
(See https://docs.ansible.com/ansible/latest/collections/community/general/cobbler_inventory.html for how `community.general.cobbler` inventory plugin is called.)

BTW, it's probably better to use `/latest/` or `/devel/` instead of `/2.10/` in these URLs. 2.10 is (very likely) no longer updated, so the new links will probably never work. With `/latest/` they will work though (once community.digitalocean 1.1.0 is included in an Ansible release).